### PR TITLE
use istio 1.1

### PR DESCRIPTION
**Pull request by lluunn**
We need 1.1 for JWT's trigger rule (excluded_paths: /healthz)

The manifest are from Istio pre-release 1.1-20190111-09-15.
Changed:
- policy is disabled instead of enabled (see README for explanation)
- istio-ingressgateway type is NodePort not LoadBalancer

/cc @kunmingg 

related: #1907

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/2259)

### CODE REVIEW

File: dependencies/istio/install/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Istio manifest for Kubeflow
 
-The manifest is taken from OSS Istio release 1.0.2.
+The manifest is taken from OSS Istio pre-release 1.1-20190111-09-15.
```
Comment (jlewi) [inline dependencies/istio/install/README.md]: Can you provide a link where these manifests are obtained?

**Comment by jlewi** [comment]
The ISTIO install still needs to be integrated into kfctl.sh? Do we have an issue tracking that?

**Comment by lluunn** [comment]
https://github.com/kubeflow/kubeflow/issues/2052 this is for installing istio with click-to-deploy

File: dependencies/istio/install/README.md (hunk 1 above)
Comment (lluunn) [reply dependencies/istio/install/README.md]: done

**Pull request merged**